### PR TITLE
add acquisition info field to each outlet

### PR DESCRIPTION
**Pull request by ClaraKuper**
based on changes made when setting up cli_device_selection

### CODE REVIEW

File: src/pupil_labs/invisible_lsl_relay/cli.py [hunk 1]
```diff
@@ -86,7 +89,10 @@ async def get_device_identifier(device_ip, device_port):
                 'is connected to the same network.'
             )
             raise exc
-        return status.phone.device_id
+        assert (
+            status.hardware.world_camera_serial
+        ), "The world camera is not connected to the device."
```
Comment (papr) [inline src/pupil_labs/invisible_lsl_relay/cli.py:94]: I feel like this should rather be a warning than an assertion. Let's fall back to `"default"`. This way, if one was to request the intrinsics for this acquisition field, one would still get a [valid response](https://api.cloud.pupil-labs.com/v2/hardware/default/calibration.v1?json). This also corresponds to the intrinsics used by the Companion app if no scene camera is connected.

File: src/pupil_labs/invisible_lsl_relay/outlets.py [hunk 2]
```diff
@@ -91,11 +114,24 @@ def pi_streaminfo(
...
     xml_channels = stream_info.desc().append_child("channels")
     [chan.append_to(xml_channels) for chan in channels]
     return stream_info
 
 
 def get_lsl_time_offset():
     return time.time() - lsl.local_clock()
+
+
+def set_acquisition_info(
```
Comment (papr) [inline src/pupil_labs/invisible_lsl_relay/outlets.py:129]: This function does not really set anything but transform data.
```suggestion
def compose_acquisition_info(
```

**Review by papr — CHANGES_REQUESTED** [review]

File: src/pupil_labs/invisible_lsl_relay/cli.py (hunk 1 above)
Comment (ClaraKuper) [reply src/pupil_labs/invisible_lsl_relay/cli.py:94]: If the camera field is none, LSL cannot write it into a string and crashes later. But I can return and empty string if the camera is not connected
.

Comment (papr) [reply src/pupil_labs/invisible_lsl_relay/cli.py:94]: I was suggesting to fallback to the literal string `"default"` if the scene camera is not connected.
```suggestion
        serial = status.hardware.world_camera_serial or "default"
        return status.phone.device_id, serial
```

Comment (ClaraKuper) [reply src/pupil_labs/invisible_lsl_relay/cli.py:94]: Ah! Sorry, I didn't get that. Sounds good!

**Review by papr — APPROVED** [review]

**Pull request merged**